README.md

```diff
@@ -28,113 +26,138 @@ The recommended way to install TorchGeo is with [pip](https://pip.pypa.io/):
+
+```python
+from pytorch_lightning import Trainer
+from torchgeo.datamodules import InriaAerialImageLabelingDataModule
+from torchgeo.trainers import SemanticSegmentationTask
+
+datamodule = InriaAerialImageLabelingDataModule(root_dir="...", batch_size=64, num_workers=6)
+task = SemanticSegmentationTask(segmentation_model="unet", encoder_weights="imagenet", learning_rate=0.1)
+trainer = Trainer(gpus=1, default_root_dir="...")
+
+trainer.fit(model=task, datamodule=datamodule)
+```
+
+<img src="https://raw.githubusercontent.com/microsoft/torchgeo/main/images/inria.png" alt="Building segmentations produced by a U-Net model trained on the Inria Aerial Image Labeling dataset"/>
 
-We provide a script, `train.py` for training models using a subset of the datasets. We do this with the PyTorch Lightning `LightningModule`s and `LightningDataModule`s implemented under the `torchgeo.trainers` namespace.
-The `train.py` script is configurable via the command line and/or via YAML configuration files. See the [conf/](conf/) directory for example configuration files that can be customized for different training runs.
+In our GitHub repo, we provide `train.py` and `evaluate.py` scripts to train and evaluate the performance of models using these datamodules and trainers. These scripts are configurable via the command line and/or via YAML configuration files. See the [conf](https://github.com/microsoft/torchgeo/blob/main/conf) directory for example configuration files that can be customized for different training runs.
 
 ```console
 $ python train.py config_file=conf/landcoverai.yaml
 ```
 
 ## Citation
 
-If you use this software in your work, please cite [our paper](https://arxiv.org/abs/2111.08872):
+If you use this software in your work, please cite our [paper](https://arxiv.org/abs/2111.08872):
 ```
 @article{Stewart_TorchGeo_deep_learning_2021,
     author = {Stewart, Adam J. and Robinson, Caleb and Corley, Isaac A. and Ortiz, Anthony and Lavista Ferres, Juan M. and Banerjee, Arindam},
     journal = {arXiv preprint arXiv:2111.08872},
     month = {11},
-    title = {{TorchGeo: deep learning with geospatial data}},
+    title = {{TorchGeo}: Deep Learning With Geospatial Data},
     url = {https://github.com/microsoft/torchgeo},
     year = {2021}
 }
 ```
 
 ## Contributing
 
-This project welcomes contributions and suggestions. If you would like to submit a pull request, see our [Contribution Guide](https://torchgeo.readthedocs.io/en/latest/user/contributing.html) for more information.
+This project welcomes contributions and suggestions. If you would like to submit a pull request, see our [Contribution Guide](https://torchgeo.readthedocs.io/en/stable/user/contributing.html) for more information.
```

Comment:
Changed all of the documentation links from "latest" to "stable" to match our new default documentation version.